deploy/deploy.gradle

```diff
@@ -43,21 +43,21 @@ afterEvaluate { project ->
         publications {
             // Creates a Maven publication called "release".
             release(MavenPublication) {
-                // Applies the component for the release build variant.
-                from components.release
-
                 // Adds Javadocs and Sources as separate jars.
                 artifact androidJavadocsJar
                 artifact androidSourcesJar
+                artifact bundleReleaseAar
 
-                groupId = "com.stripe"
-                artifactId = "stripe-android"
+                groupId = GROUP
```

Comment:
I think I had issues pulling `GROUP` from `gradle.properties`. Can we leave it as it was or pull it from the `project`?

Reply:
hmm could you clarify what issues it is? we are reading `VERSION_NUMBER` from `gradle.properties` now too, I wonder if we should hardcode that too

Reply:
defined `group_name` and `version_name` in root `build.gradle`, `GROUP` is also used there, so I think this should avoid any possible issues